packages/unlock-js/openapi.yml

```diff
@@ -2676,6 +2676,20 @@ paths:
           schema:
             type: number
 
+        - in: query
+          name: lockAddress
+          required: false
+          description: The address of the lock
+          schema:
+            type: string
+
+        - in: query
+          name: keysToPurchase
```

Comment:
need this to know how many keys we are purchasing in order to have the total price to charge in `USD`